signer-service/src/api/controllers/moonbeam.controller.js

```diff
@@ -66,12 +68,16 @@ exports.sendStatusWithPk = async () => {
     const balance = await publicClient.getBalance({ address: moonbeamExecutorAccount.address });
 
     // We are checking if the balance is less than 10 GLMR
-    const minimum_balance = Big(MOONBEAM_FUNDING_AMOUNT_UNITS).times(Big(10).pow(18));
-    if (balance < minimum_balance) {
+    const minimumBalance = Big(MOONBEAM_FUNDING_AMOUNT_UNITS).times(Big(10).pow(18));
+
+    if (balance < minimumBalance) {
+      slackService.sendMessage({
+        text: `Current balance of funding account is ${balance} GLMR please charge this account ${moonbeamExecutorAccount.address}.`,
```

Comment:
`balance` is the raw amount, not the unit amount, correct? In that case can we write
```suggestion
        text: `Current balance of funding account is ${balance.div(Big(10).pow(18)).toFixed(2,0)} GLMR please charge this account ${moonbeamExecutorAccount.address}.`,
```